src/Utility/Assembly.h

```diff
@@ -61,7 +61,7 @@ class Assembly {
     llvm::MCDisassembler::DecodeStatus getInstruction(llvm::MCInst &inst, uint64_t &size,
                                             llvm::ArrayRef<uint8_t> bytes, uint64_t address) const;
 
-    void printDisasm(const llvm::MCInst &inst, llvm::raw_ostream &out = llvm::errs()) const;
+    void printDisasm(const llvm::MCInst &inst, uint64_t address, llvm::raw_ostream &out = llvm::errs()) const;
```

Comment:
``uintptr_t`` instead of of ``uint64_t``

Reply:
I prefer to keep a ``uint64_t`` to match the LLVM API (https://github.com/llvm/llvm-project/blob/llvmorg-10.0.1/llvm/include/llvm/MC/MCDisassembler/MCDisassembler.h#L77)